geode-core/src/main/java/org/apache/geode/internal/statistics/HostStatHelper.java

```diff
@@ -31,109 +30,55 @@
  * Provides native methods which fetch operating system statistics.
  */
 public class HostStatHelper {
-  private static final int LINUX_CODE = 3; // x86 Linux
-
   private static final int PROCESS_STAT_FLAG = 1;
   private static final int SYSTEM_STAT_FLAG = 2;
 
-  private static final int osCode;
-
   static {
     String osName = System.getProperty("os.name", "unknown");
-    if (!PureJavaMode.osStatsAreAvailable()) {
```

Comment:
I removed the `osStatsAreAvailable()` called because is redundant: if it returns `false`, that means that `osName.startsWith("Linux")` is `false`, which is what will be checked now.